react/features/polls/components/native/PollCreate.js

```diff
@@ -148,7 +154,8 @@ const PollCreate = (props: AbstractProps) => {
                     keyExtractor = { (item, index) => index.toString() }
                     ref = { answerListRef }
                     renderItem = { renderListItem } />
-                <View style = { chatStyles.pollCreateButtonsContainer }>
+                <KeyboardAvoidingView
```

Comment:
This is optional regarding the fact that polls screen will or should be redesigned because there too many options for a small screen like that.